scripts/templates/detector.tf.j2

```diff
@@ -77,7 +77,7 @@ resource "signalfx_detector" "{{ id }}" {
         {%- else %}
           {%- set negative_comparator = reverse_comparator ~ '=' %}
         {%- endif %}
-    detect(when(signal {{ rule.comparator }} ${var.{{ id }}_threshold_{{ severity }}})
+    detect(when(signal {{ rule.comparator }} ${var.{{ id }}_threshold_{{ severity }}}{%- if rule.lasting is string %}, lasting=${var.{{ id }}_lasting_{{severity}}}{%- endif %})
```

Comment:
why not adding the `at_least` also ?

like for: 

https://github.com/claranet/terraform-signalfx-detectors/blob/master/modules/integration_aws-elb/detectors-elb.tf#L110

see https://developers.signalfx.com/signalflow_analytics/functions/lasting_function.html 

we can set it to `1` if not defined in yaml config

Reply:
Instead of forcing to 1, I've used the same hack for lasting with a condition in the Jinja template.